core/src/main/java/org/ehcache/CacheManagerBuilder.java

```diff
@@ -48,13 +48,11 @@ public T build() {
 
   public static CacheManager newCacheManager(final Configuration configuration) {
```

Comment:
These methods are more `factory` methods than a builder pattern. I propose renaming `CacheManagerBuilder` to something else - `CacheManagerFactory`?